.github/workflows/build.yml

```diff
@@ -31,27 +27,6 @@ jobs:
         run: dotnet restore
 
       - name: 🛠️ Building library in release mode
```

Comment:
We want to maintain the tasks:

          - name: 🛒 Checkout repository
        uses: actions/checkout@v4
        with:
          fetch-depth: 0

      - name: ⚙️ Setup dotnet 8.0.x
        uses: actions/setup-dotnet@v4
        with:
          dotnet-version: '8.0.x'

      - name: 🧹 Clean
        run: dotnet clean -c Release && dotnet nuget locals all --clear

      - name: 🔁 Restore packages
        run: dotnet restore